src/attack.c

```diff
@@ -1421,8 +1421,15 @@ attack_network (struct scan_globals *globals, kb_t *network_kb)
   plugins_scheduler_free (sched);
 
   gettimeofday (&now, NULL);
-  g_message ("Total time to scan all hosts : %ld seconds",
-             now.tv_sec - then.tv_sec);
+  if (alive_hosts_list)
+    g_message ("Vulnerability scan %s finished in %ld seconds: "
+               "%d alive hosts of %d",
+               globals->scan_id, now.tv_sec - then.tv_sec,
+               gvm_hosts_count (alive_hosts_list), gvm_hosts_count (hosts));
```

Comment:
If consider alive was chosen and e.g. 10 hosts are in the target range. This will print `... :  10 alive hosts of 10` even though they might all be dead. Maybe for the logs the message is good enough though.

Reply:
This is surely debatable. The "consider alive" option is special anyway. I suggest
to leave as is.